monkey/common/agent_configuration/agent_sub_configurations.py

```diff
@@ -50,7 +50,15 @@ class ScanTargetConfiguration(MutableInfectionMonkeyBaseModel):
                                      Example: ("1.1.1.1", "2.2.2.2/24", "myserver")
     """
 
-    scan_my_networks: bool = Field(title="Scan Agent's networks", default=False)
+    scan_my_networks: bool = Field(
+        title="Scan Agent's networks",
+        default=False,
+        configuration_message=(
```

Comment:
Why not have this field called "warning_message". We could also support "info_message". This would allow a field to provide one or both.